automated/linux/kselftest/kselftest.sh

```diff
@@ -107,6 +108,20 @@ while getopts "i:n:c:T:t:s:u:p:L:S:b:g:e:h" opt; do
         e)
             export ENVIRONMENT="${OPTARG}"
             ;;
+        E)
+            if [ -n "$OPTARG" ]; then
+                 IFS='=' read -r env_name env_val << EOF
+$OPTARG
+EOF
+                if [ -z "$env_name" ] || [ -z "$env_val" ]; then
```

Comment:
is there a reason for "$env_name" here and not "${env_name}" but doing that further down?